src/services/mongoose/users-ips.ts

```diff
@@ -56,7 +56,7 @@ export const UsersIpRepository = (): IUsersIPsRepository => {
 
 const userIPsFromRaw = (result: UserIPsType): UserIPs => {
   return {
-    id: result.id as UserId,
+    id: fromObjectId<UserId>(result._id) as UserId,
```

Comment:
casting `as UserId` shouldn't be needed. That is what the `<UserId>` generic is for.